src/lib/transition_handler/processor.ml

```diff
@@ -93,7 +93,9 @@ module Make (Inputs : Inputs.S) :
                        Transition_frontier.add_breadcrumb_exn frontier
                          breadcrumb ;
                        Writer.write processed_transition_writer transition ;
-                       Catchup_scheduler.notify catchup_scheduler ~transition
+                       Deferred.return
```

Comment:
Why do the `bind` above then? It would be better to just perform a map.

Reply:
If bind means Deferred.bind then yes, but here the bind is actually Deferred.Result.bind.